website/docs/index.html.markdown

```diff
@@ -73,11 +73,12 @@ resource "newrelic_alert_policy_channel" "alert_email" {
-- `infra_api_url` - (Optional) Possibility to change the Infra API url (default is https://infra-api.newrelic.com/v2). This is for instance relevant if the New Relic account is in the EU. The Infra API url must then be set to https://infra-api.eu.newrelic.com/v2. Can also use `NEWRELIC_INFRA_API_URL` environment variable.
+- `api_key` - (Required except for `newrelic_insights_event` resource) Your New Relic API key. The `NEWRELIC_API_KEY` environment variable can also be used.
+- `api_url` - (Optional) This argument changes the main REST API URL (default is https://api.newrelic.com/v2). If the New Relic account is in the EU, the API URL must be set to https://api.eu.newrelic.com/v2. The `NEWRELIC_API_URL` environment variable can also be used.
+- `synthetics_api_url` - (Optional) This argument changes the Synthetics API URL (default is https://synthetics.newrelic.com/synthetics/api/v3). If the New Relic account is in the EU. the API URL must be set to https://synthetics.eu.newrelic.com/synthetics/api/v3. The `NEWRELIC_SYNTHETICS_API_URL` environment variable can also be used.  This URL is used to provision Synthetics monitors and monitor scripts only.
+- `infra_api_url` - (Optional) This argument changes the Infrastructure API URL (default is https://infra-api.newrelic.com/v2). If the New Relic account is in the EU, the Infra API URL must be set to https://infra-api.eu.newrelic.com/v2. The `NEWRELIC_INFRA_API_URL` environment variable can also be used.  This URL is used to provision Infrastructure alert conditions only.
 - `insecure_skip_verify` - (Optional) Trust self-signed SSL certificates. If omitted, the `NEWRELIC_API_SKIP_VERIFY` environment variable is used.
 - `insights_account_id` - (Optional) Your New Relic Account ID used when inserting Insights events via the `newrelic_insights_event` resource. Can also use `NEWRELIC_INSIGHTS_ACCOUNT_ID` environment variable.
 - `insights_insert_key` - (Optional) Your Insights insert key used when inserting Insights events via the `newrelic_insights_event` resource. Can also use `NEWRELIC_INSIGHTS_INSERT_KEY` environment variable.
-- `insights_insert_url` - (Optional) Possibility to change the Insights insert url (default is https://insights-collector.newrelic.com/v1/accounts). This is for instance relevant if the New Relic account is in the EU. The Insights API url must then be set to https://insights-collector.eu.newrelic.com/v1. Can also use `NEWRELIC_INSIGHTS_INSERT_URL` environment variable.
+- `insights_insert_url` - (Optional) Possibility to change the Insights insert URL (default is https://insights-collector.newrelic.com/v1/accounts). This is for instance relevant if the New Relic account is in the EU. The Insights API URL must then be set to https://insights-collector.eu.newrelic.com/v1. Can also use `NEWRELIC_INSIGHTS_INSERT_URL` environment variable.
```

Comment:
Not a blocker by any means, but we could possibly remove **"for instance"** from the "This is for instance relevant if the New Relic account..." sentence.

Reply:
good catch.